File: sync/rest/maps/query-map/query-map.5.x.cs
// Download the twilio-csharp library from twilio.com/docs/libraries/csharp
using System;
using Twilio;
using Twilio.Rest.Preview.Sync.Service.SyncMap;

public class Example
{
    public static void Main(string[] args)
    {
        // Find your Account SID and Auth Token at twilio.com/console
        const string accountSid = "ACXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX";
        const string authToken = "your_auth_token";
        const string serviceSid = "ISXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX";

        TwilioClient.Init(accountSid, authToken);

        var items = SyncMapItemResource.Read(
            serviceSid,
            "Players",
            from: "steph_curry",
            order: SyncMapItemResource.QueryResultOrderEnum.Asc);

        foreach (var item in items)
        {
            Console.WriteLine($"{item.Key} = {item.Data}");
        }
    }
}

File: sync/rest/maps/query-map/query-map.5.x.rb
require 'twilio-ruby'

# Initialize the client
# To set up environmental variables, see http://twil.io/secure
account_sid = ENV['TWILIO_ACCOUNT_SID']
auth_token = ENV['TWILIO_AUTH_TOKEN']
client = Twilio::REST::Client.new(account_sid, auth_token)

# Create the service
service = client.sync.v1.services('ISXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX')

# Query for a map item
items = service.sync_maps('Players').sync_map_items.stream(
  from:  'steph_curry',
  order: 'asc'
)
items.each do |map_item|
  puts map_item.key
end
